CMakeLists.txt

```diff
@@ -78,7 +83,8 @@ set(LIBRARY_TARGET_NAME OsqpEigen)
 set(${LIBRARY_TARGET_NAME}_SRC
   src/Data.cpp
   src/Settings.cpp
-  src/Solver.cpp)
+  src/Solver.cpp
+  src/Debug.cpp)
```

Comment:
The header file should be included in the ``${LIBRARY_TARGET_NAME}_HDR`` list as well.